packages/web-pkg/src/composables/appDefaults/useAppNavigation.ts

```diff
@@ -14,20 +15,43 @@ export interface AppNavigationResult {
   closeApp(): void
 }
 
+const paramsKey = 'params'
```

Comment:
I don't mind whether we stick to the short `params` or go for a more semantic/explicit `contextRouteParams` or `contextParams`.

Reply:
Looks a bit verbose, but I actually prefer `contextRouteParams` as it'd be consistent with the `contextRouteName` like you already mentioned